src/foundation.cs

```diff
@@ -12712,30 +12712,18 @@ interface NSPropertyListSerialization {
 		bool IsValidForFormat (NSObject plist, NSPropertyListFormat format);
 	}
 
-#if XAMCORE_2_0 || !MONOMAC
 	interface INSExtensionRequestHandling { }
 
-	[iOS (8,0)][Mac (10,10)] // Not defined in 32-bit
-	[Protocol, Model]
-	[BaseType (typeof (NSObject))]
-	interface NSExtensionRequestHandling {
-		[Abstract]
-		[Mac (10,10, onlyOn64 : true)] 
-		// @required - (void)beginRequestWithExtensionContext:(NSExtensionContext *)context;
-		[Export ("beginRequestWithExtensionContext:")]
-		void BeginRequestWithExtensionContext (NSExtensionContext context);
-	}
-#else
 	[iOS (8,0)][Mac (10,10, onlyOn64:true)] // Not defined in 32-bit
 	[Protocol, Model]
 	[BaseType (typeof (NSObject))]
 	interface NSExtensionRequestHandling {
 		[Abstract]
+		[Mac (10,10, onlyOn64 : true)] 
```

Comment:
^ not needed, the type already has the same attribute

Reply:
@spouliot  fixed: a2ffd585f70492620a2f34c6ca89f674eb3e7fe0